python-daemon/setup.py

```diff
@@ -75,6 +75,8 @@
     'idna>=2.5',
     'bleach>=1.5.0',
     'pyspark',
+    'tensorflow==2.3',
```

Comment:
This dependency is cause error during test: https://app.travis-ci.com/github/apache/incubator-marvin/jobs/536187758
Might need to point the `pip` to the correct artifact.

Reply:
Also, we also see `This package requires Rust >=1.41.0.` in the error log, and is requesting us to run `pip install --upgrade pip`.
We should simply upgrade the `pip` in the daemon and both error should be solved.